.vscode/launch.json

```
@@ -176,6 +176,21 @@
       "outFiles": [
         "${workspaceFolder}/../.js"
       ]
+    },
+    {
+      "name": "Debug selected system test file with Playwright",
```

The naming is a big long, and does not follow the format of other debug configurations we have.
We might want to simplify similarly to `Run Mocha Test`:

```suggestion
      "name": "Run Playwright Test",
```